UnitystationLauncher/Services/InstallationService.cs

```diff
@@ -70,18 +78,30 @@ public List<Installation> GetInstallations()
             && d.BuildVersion == buildVersion);
     }
 
-    public (Download?, string) DownloadInstallation(Server server)
+    public async Task<(Download?, string)> DownloadInstallation(Server server)
     {
         string? downloadUrl = server.GetDownloadUrl(_environmentService);
         if (string.IsNullOrWhiteSpace(downloadUrl))
         {
             const string failureReason = "Empty or missing download url for server.";
             Log.Warning(failureReason + $" ServerName: {server.ServerName}");
-            return (null, failureReason);
+            return (null!, failureReason);
         }
 
-        Download? download = GetInProgressDownload(server.ForkName, server.BuildVersion);
+        server.ServerGoodFileVersion = "1.0.0"; //TODO
```

Comment:
Is this TODO done?

Reply:
well have to wait for the hub to be set up to have that information

Reply:
Ah, sorry I didn't realize that part wasn't done yet. If this is merged before thats done we can just create an issue for it.